test/e2e/sched/utils/node.go

```diff
@@ -22,17 +22,30 @@ import (
 	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	"github.com/k8stopologyawareschedwg/deployer/pkg/deployer/platform"
+	"github.com/openshift-kni/numaresources-operator/test/utils/configuration"
 )
 
-func ListMasterNodes(aclient client.Client) ([]corev1.Node, error) {
+const (
+	// LabelMasterRole contains the key for the role label
+	LabelMasterRole = "node-role.kubernetes.io/master"
+
+	// LabelControlPlane contains the key for the control-plane role label
+	LabelControlPlane = "node-role.kubernetes.io/control-plane"
+)
 
+func ListMasterNodes(aclient client.Client) ([]corev1.Node, error) {
 	nodeList := &corev1.NodeList{}
 	labels := metav1.LabelSelector{
 		MatchLabels: map[string]string{
-			"node-role.kubernetes.io/control-plane": "",
-			"node-role.kubernetes.io/master":        "",
+			LabelMasterRole: "",
 		},
 	}
+	if configuration.Platform == platform.Kubernetes {
```

Comment:
this is going to change when OCP catches up with kube, but it's good enough for now